### PR TITLE
Adding landing page.

### PR DESCRIPTION
**Pull request by bipin7yadav**
landing-page

### CODE REVIEW

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import "./Footer.css"
+function Footer(){
+    return<>
+    <footer className="footer-containet">
```
Comment (MarkVed17) [inline src/components/Footer/Footer.jsx:4]: Typo
Doesn't affect the code as the CSS class is given the same name but it must be footer-container*

File: src/pages/Home/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    
+            <div class="con1">
+    
+                <div class="img-con">
+                    <img src="https://assets.myntassets.com/f_webp,dpr_1.5,q_60,w_210,c_limit,fl_progressive/assets/images/2164410/2018/6/19/f6154681-bd6b-46d5-83ff-9b31d059f2e81529387774162-Roadster-Time-Travlr-Men-Navy--Grey-Regular-Fit-Checked-Casu-1.jpg" alt="./" class="img"/>
+                    <label class="label">Shirts</label>
+    
+                </div>
+    
+                <div class="img-con">
```
Comment (MarkVed17) [inline src/pages/Home/Home.jsx:27]: Suggestion: This could be turned into a reusable component.

**Review by MarkVed17 — APPROVED** [review]
Everything looks fine.
Just keep a check on your CSS conventions, code formatting & classNames.
Good to merge.

File: src/App.js [hunk 3]
```diff
@@ -1,41 +1,24 @@
...
-      </header>
+      <Navigation/>
+      <Routes>
+        <Route path="/" element={<Home/>} />
+        <Route path="/Auth" element={<Auth/>}/>
+        <Route path="/AllProducts" element={<AllProducts/>} />
+        <Route path="/MyCart" element={<MyCart/>} />
+        <Route path="/SingleProduct" element={<SingleProduct/>} />
+        <Route path="/Wishlist" element={<Wishlist/>} />
+      </Routes>
```
Comment (himadri2110) [inline src/App.js:19]: Suggestion: You can make a separate component for Routes, this will keep App.js clean and short

**Review by himadri2110 — APPROVED** [review]
Good to go.

**Pull request merged**